### DISCUSSION TITLE
how to configure renovate to ignore "illegal characters"

### DISCUSSION THREAD

**Discussion by bloodsper** [renovatebot/renovate#13015] [Request Help]
We have a template repo and want Renovate to keep those template dependencies up to date so there isn't 50 PRs off the back when a developer creates a spin-off

The issue is the yarn.lock and package.lock path is `{{ template }}/yarn.lock` and that spins up an artifact error comment and status check. I was able to ignore the commend, but the status check is still giving the PR a big ❌ 

Any ways around this

**Comment by rarkins** [comment]
Can you create a reproduction of this in a public repo? I don't think I understand what you mean. Alternatively, you can try expanding your description.
